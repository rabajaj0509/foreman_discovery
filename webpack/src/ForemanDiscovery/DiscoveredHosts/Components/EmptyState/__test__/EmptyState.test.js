import { testComponentSnapshotsWithFixtures } from '@theforeman/test';

import ForemanEmptyState from '../EmptyState';

const fixtures = {
  'render with Props': { docUrl: 'https://foreman.example.com' },
};

describe('ForemanEmptyState', () => {
  describe('rendering', () =>
    testComponentSnapshotsWithFixtures(ForemanEmptyState, fixtures));
});
