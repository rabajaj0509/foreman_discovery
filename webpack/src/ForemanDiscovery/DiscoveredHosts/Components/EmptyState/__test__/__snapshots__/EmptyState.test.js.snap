// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForemanEmptyState rendering render with Props 1`] = `
<DefaultEmptyState
  description="No discovered hosts found in this context. This page shows discovered bare-metal or virtual nodes waiting to be provisioned."
  documentation={
    Object {
      "url": "https://foreman.example.com",
    }
  }
  header="Foreman Discovery"
  icon="gears"
  iconType="fa"
  secondaryActions={Array []}
/>
`;
